### DISCUSSION TITLE
Subtype'ların Card'larda Gösterilmesi

### DISCUSSION THREAD

**Discussion by okazkayasi** [acikkaynak/eczane-frontend#124] [Ideas]
@unitythemaker tarafindan açıldı. Mesajı:

Data with different subtypes are almost impossible to seperate. By subtypes I mean like: Regular hospital and field hospital.

![image](https://user-images.githubusercontent.com/10429728/218779600-dfa5f576-c7a0-4acc-886f-87eec2635b94.png)

**Comment by yagmurty** [comment]
Design'da da bu konu var ileteceğim.
